apps/fishing-map/pages/api/track-corrections/[workspaceId]/index.ts

```diff
@@ -13,6 +18,49 @@ export type GetAllIssuesAPIResponse = TrackCorrection[]
 
 export type APIResponse = ErrorAPIResponse | GetAllIssuesAPIResponse | CreateIssueAPIResponse
 
+const handlers = {
+  GET: async (req: NextApiRequest, res: NextApiResponse<APIResponse>, workspaceId: string) => {
+    const issues = await getWorkspaceIssues(workspaceId)
+    return res.status(200).json(issues)
+  },
+
+  POST: async (req: NextApiRequest, res: NextApiResponse<APIResponse>, workspaceId: string) => {
+    try {
+      const body = req.body
+
+      if (body.issueBody && body.commentBody) {
+        await createNewIssue(
+          body.issueBody as TrackCorrection,
+          body.commentBody as TrackCorrectionComment,
+          workspaceId
+        )
+        return res.status(201).json(body.issueBody as TrackCorrection)
+      } else if (body.issueId && body.commentBody) {
```

Comment:
Following the REST standard, I think this needs to be moved to the following [endpoint](https://github.com/GlobalFishingWatch/frontend/blob/ui-and-spreadsheet/apps/fishing-map/pages/api/track-corrections/%5BworkspaceId%5D/%5BissueId%5D.ts#L24) 
`api/track-corrections/[workspaceId/[issueId]` 

Because we are modifying the content of an issue and using the same endpoint for creating an issue or modifying one is too confusing